src/app.py

```diff
@@ -1,13 +1,25 @@
 import time
+import eel
```

Comment:
Could we be a little more clear about how these imports come together? Maybe model them, or comments?

Reply:
This is a good idea. Some diagramming this would be great to do.